### ISSUE TITLE
Better documentation for using this library in an imperative way

### ISSUE THREAD

**Issue by danielr1996** [authts/react-oidc-context#511]
The documentation states that for using this library in an imperative way (e.g. for using with redux) one should configure the `oidc-client-ts` library with the `WebStorageStateStore` option and that the needed information then can be found in `localStorage.getItem(`oidc.user:<your authority>:<your client id>`)`

However I observed that I didn't need to use the `WebStorageStateStore` config and had to retrieve the information from session storage rather than local storage. I don't know if that holds true in any case or just in my case so I'm not confident enough to submit a PR for the documentation, but perhaps someone with a deeper understanding of how this works can expand the documentation to be more clear on how to do this.

**Comment by pamapa** [comment]
You are right that can be improved. At the end it does not depend on redux, but on how you have configured the underlying `oidc-client-ts` library (settings.userStore).

Fell free to make a MR to improve that doc section...
